docs/provider/daemon/deposit_funds.md

```diff
@@ -173,6 +173,10 @@ This updates the current market percentage fee to 1% (by default it's 0.25%).
 $ tdex fixedfee --base_fee 600 --quote_fee 20000
 ```
 
+:::tip
+Consider to set a fixed fee of ~650 sats of LBTC and correspond amount of quote asset on the markets of your daemon since you'll always pay for network fees for the all the swap transactions made with traders.
```

Comment:
```suggestion
Consider to set a fixed fee of ~650 sats of LBTC and correspond amount of quote asset on the markets of your daemon since you'll always pay for network fees for the all the swap transactions made by traders.
```